compiler/circle-mpqsolver/src/CircleMPQSolver.cpp

```diff
@@ -113,9 +111,11 @@ int entry(int argc, char **argv)
   {
     // The third parameter of setenv means REPLACE.
     // If REPLACE is zero, it does not overwrite an existing value.
-    setenv("LUCI_LOG", "100", 0);
+    setenv("LUCI_LOG", "1", 0);
```

Comment:
Q) why change 100 to 1 ?

Reply:
> Q) why change 100 to 1 ?

Because `quantizer` will add its own `vebose` output and the whole output will be very verbose in my personal opinion.
@seanshpark @jinevening 
Should i set `LUCI_LOG` to `100`?

Reply:
> Because quantizer will add its own vebose output

Can you point out which source code?

> Should i set LUCI_LOG to 100?

Um? I am confused why you ask about this change if you know what you are doing.
Or you don't know the meaning of this change?

Reply:
I agree that `LUCI_LOG = 100` will print too much logs.

@stamalakhov  I think there are two different types of log messages.

1. Always printed (this has to be implemented with `std::cout`, not `VERBOSE` logger).
2. Printed only when `LUCI_LOG` is non-zero (this has to be implemented with luci logger).

Could you keep the verbose level, and change `VERBOSE(l, 0)` to `std::cout` for always-printed messages?

```
$ git grep VERBOSE
src/CircleMPQSolver.cpp:  VERBOSE(l, 0) << ">> Searching mixed precision configuration " << std::endl
src/CircleMPQSolver.cpp:        VERBOSE(l, 0) << "algorithm: bisection (auto)" << std::endl;
src/CircleMPQSolver.cpp:        VERBOSE(l, 0) << "algorithm: bisection (Q16AtFront)";
src/CircleMPQSolver.cpp:        VERBOSE(l, 0) << "algorithm: bisection (Q8AtFront)";
src/CircleMPQSolver.cpp:    VERBOSE(l, 0) << "qerror metric: MAE" << std::endl
src/CircleMPQSolver.cpp:      VERBOSE(l, 0) << "Saving output model to " << output_model_path << std::endl;
src/bisection/BisectionSolver.cpp:  VERBOSE(l, 0) << std::endl << ">> Running bisection(auto) algorithm" << std::endl;
src/bisection/BisectionSolver.cpp:  VERBOSE(l, 0) << "Qerror of front half: " << error_at_input << std::endl;
src/bisection/BisectionSolver.cpp:  VERBOSE(l, 0) << "Qerror of rear half: " << error_at_output << std::endl;
src/bisection/BisectionSolver.cpp:    VERBOSE(l, 0) << "Front part will be Q16, while the rear will be Q8" << std::endl;
src/bisection/BisectionSolver.cpp:    VERBOSE(l, 0) << "Front part will be Q8, while the rear will be Q16" << std::endl;
src/bisection/BisectionSolver.cpp:  VERBOSE(l, 0) << std::endl << ">> Computing baseline qerrors" << std::endl;
src/bisection/BisectionSolver.cpp:  VERBOSE(l, 0) << "Full int16 model qerror: " << int16_qerror << std::endl;
src/bisection/BisectionSolver.cpp:  VERBOSE(l, 0) << "Full uint8 model qerror: " << uint8_qerror << std::endl;
src/bisection/BisectionSolver.cpp:  VERBOSE(l, 0) << "Target qerror: " << _qerror << std::endl;
src/bisection/BisectionSolver.cpp:      VERBOSE(l, 0) << "Front part will be Q16, while the rear will be Q8" << std::endl;
src/bisection/BisectionSolver.cpp:      VERBOSE(l, 0) << "Front part will be Q8, while the rear will be Q16" << std::endl;
src/bisection/BisectionSolver.cpp:  VERBOSE(l, 0) << std::endl;
src/bisection/BisectionSolver.cpp:    VERBOSE(l, 0) << "Looking for the optimal configuration in [" << min_depth << " , " << max_depth
src/bisection/BisectionSolver.cpp:      VERBOSE(l, 0) << "Qerror at depth " << cut_depth << " is " << cur_accuracy
src/bisection/BisectionSolver.cpp:      VERBOSE(l, 0) << "Qerror at depth " << cut_depth << " is " << cur_accuracy
src/bisection/BisectionSolver.cpp:  VERBOSE(l, 0) << "Found the best configuration at depth " << best_depth << std::endl;
```

Reply:
> Can you point out which source code?

@seanshpark circle mpqsolver internally invokes circle quantizer. Setting LUCI_LOG=100 will turn on all levels of logs inside circle-quantizer, so the final log will be too verbose.

Reply:
> Could you keep the verbose level, and change `VERBOSE(l, 0)` to `std::cout` for always-printed messages?

@jinevening 
Yes. I'll change log level for `verbose` mode to be `100` and will introduce always printed messages in the next PR.

Reply:
@jinevening 
done.